bin/jobsub_cmd

```diff
@@ -107,10 +109,10 @@ def main() -> None:
 
     cmd = cmd.replace("jobsub_", "condor_")
 
-    execargs.insert(0, cmd)
+    execargs.insert(0, f"/usr/bin/{cmd}")
```

Comment:
Surprisingly, yes... We used to exec() the program, now we os.system it, so that our span covers running the program.  I'm pretty sure if we exec() the program, we never end the trace.